x-pack/packages/security-solution/navigation/src/constants.ts

```diff
@@ -15,3 +15,88 @@ export enum LinkCategoryType {
   accordion = 'accordion',
   separator = 'separator',
 }
+
+/**
+ * External (non-Security) page names that need to be linked in the Security navigation.
+ * Format: `<pluginId>:<deepLinkId>/<path>`.
+ *
+ * `pluginId`: is the id of the plugin that owns the deep link
+ *
+ * `deepLinkId`: is the id of the deep link inside the plugin.
+ * Keep empty for the root page of the plugin, e.g. `osquery:`
+ *
+ * `path`: is the path to append to the plugin and deep link.
+ * This is optional and only needed if the path is not registered in the plugin's `deepLinks`. e.g. `integrations:/browse/security`
+ * The path should not be used for links displayed in the main left navigation, since highlighting won't work.
+ **/
+export enum ExternalPageName {
```

Comment:
This enum has been moved from _security_solution_serverless_ plugin to the security navigation package, so we can import and use it from any plugin.